iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DeviceMethodTests.java

```diff
@@ -47,16 +45,11 @@ public DeviceMethodTests(IotHubClientProtocol protocol, AuthenticationType authe
         super(protocol, authenticationType, clientType, publicKeyCert, privateKey, x509Thumbprint);
     }
 
-    @Before
-    public void cleanToStart() throws Exception
-    {
-        super.cleanToStart();
```

Comment:
This creates a device/module client for this test and opens it. Not every test needs this though, so I moved this logic to the ```super.openDeviceClientAndSubscribeToMethods()``` call instead